apps/crn-server/test/data-providers/contentful/reminders.data-provider.test.ts

```diff
@@ -0,0 +1,1264 @@
+            );
+
+            const eventMockResponse =
+              getContentfulReminderEventsCollectionItem();
+            eventMockResponse!.startDate = startDate;
+            eventMockResponse!.endDate = endDate;
+
+            const users = getContentfulReminderUsersContent();
+            users!.role = asapRole;
+            users!.teamsCollection!.items[0]!.role = teamRole;
+
+            contentfulGraphqlClientMock.request.mockResolvedValueOnce({
+              eventsCollection: {
+                items: [eventMockResponse],
+              },
+              users,
+            });
+
+            contentfulGraphqlClientMock.request.mockResolvedValueOnce(
+              getTeamProjectManagerResponse(),
+            );
+
+            const result = await remindersDataProvider.fetch(
+              fetchRemindersOptions,
+            );
+
+            const sharePresentationReminder = getSharePresentationReminder();
+            sharePresentationReminder.data.endDate = endDate;
+            sharePresentationReminder.data.pmId = 'project-manager-1';
+            expect(result).toEqual({
+              total: 1,
+              items: [sharePresentationReminder],
+            });
+          },
+        );
+
+        it.each`
+          asapRole     | teamRole
+          ${`Grantee`} | ${`Project Manager`}
+          ${`Grantee`} | ${`ASAP Staff`}
+          ${`Staff`}   | ${`Lead PI (Core Leadership)`}
+          ${`Staff`}   | ${`Lead PI (Core Leadership)`}
+          ${`Staff`}   | ${'Co-PI (Core Leadership)'}
+          ${`Staff`}   | ${'Collaborating PI'}
+          ${`Staff`}   | ${`Key Personnel`}
+          ${`Staff`}   | ${`Scientific Advisory Board`}
+          ${`Staff`}   | ${`Project Manager`}
+          ${`Staff`}   | ${`ASAP Staff`}
+        `(
+          `Should not fetch the reminder when user has asap role $asapRole and team role $teamRole`,
+          async ({ asapRole, teamRole }) => {
+            const startDate = '2023-01-01T08:00:00Z';
+            const endDate = '2023-01-01T10:00:00Z';
+            jest.setSystemTime(
+              DateTime.fromISO('2023-01-02T09:00:00Z').toJSDate(),
+            );
+
+            const eventMockResponse =
+              getContentfulReminderEventsCollectionItem();
+            eventMockResponse!.startDate = startDate;
+            eventMockResponse!.endDate = endDate;
+
+            const users = getContentfulReminderUsersContent();
+            users!.role = asapRole;
+            users!.teamsCollection!.items[0]!.role = teamRole;
+
+            contentfulGraphqlClientMock.request.mockResolvedValueOnce({
+              eventsCollection: {
+                items: [eventMockResponse],
+              },
+              users,
+            });
+
+            const result = await remindersDataProvider.fetch(
+              fetchRemindersOptions,
+            );
+
+            expect(result.items.map((r) => r.type)).not.toContain(
```

Comment:
you sometimes use this pattern for your expectation but the other time you just assert that no results were returned. I would lean towards the second one although it might leave less room for extension later, question is if maybe you should only use one of the two?

Reply:
I prefer to use `not.toContain` because sometimes like in the test below, I am testing the this type of reminder shouldn't appear if the event hasn't ended. But given the conditions, other type of reminder could appear, like "Happening Now". So the response does not come empty
![Screenshot 2023-06-29 at 09 37 51](https://github.com/yldio/asap-hub/assets/16595804/7a9e3ee9-5b41-40f2-8bd8-023a8984d17b)

I've changed all the empty responses to `not.toContain` [review: change expect empty result to not.toContain reminder type](https://github.com/yldio/asap-hub/pull/3236/commits/db37835885729c3e4c544dffc9a4b813db85d695)